src/veupath/redmine/client/checks/check_genome_issues.py

```diff
@@ -280,7 +284,7 @@ def main():
     elif args.check:
         check_genome_issues(issues)
     elif args.report:
-        report_genome_issues(issues, args.report)
+        report_genome_issues(issues, args.report, ensembl_version=args.ensembl_version)
```

Comment:
I may be wrong, but unless you define a default value for this, it should raise an exception if `--ensembl_version` is not defined in the command line.

Reply:
It does work and I don't see mypy/pylint complain :/